docs/modules/publisher.md

```diff
@@ -258,6 +258,17 @@ To ensure operation of OPC Publisher over restarts, it's required to map configu
 
 In version 2.6 and above, username and password are stored in plain text in the configuration file. It must be ensured that the configuration file is protected by the file system access control of the host file system. The same must be ensured for the file system based certificate store, since it contains the private certificate and private key of OPC Publisher.
 
+### Use customer certs in OPC Publisher
```

Comment:
maybe we can use `custom OPC UA application instance certificate` instead `customer certs`. I think it would be good to emphasize a bit more that we're talking about OPC UA Client certificate here.

Reply:
changed